certbot/certbot/main.py

```diff
@@ -0,0 +1,14 @@
+"""Certbot main public entry point."""
+from certbot._internal import main as internal_main
+
+
+def main(cli_args=None):
+    """Command line argument parsing and main script execution.
```

Comment:
I'd like to improve these docs. (Should `cli_args` include `sys.argv[0]`? What return values can be expected?)

Since this was just copied from the internal function though, I'm fine doing it later.

Reply:
I literally don't know the answers to those questions, since, as you correctly identified, I just copied it from the existing comment.